### ISSUE TITLE
Option to move between nodes and chunks

### ISSUE THREAD

**Issue by evo-za** [alicevision/meshroom#1100]
It would be nice to have the option for the program to automatically highlight the currently processing node, and then if the user is in the log tab of that node, automatically move between chunks. That way one can sit back and monitor the progress without clicking between nodes and chunks.

**Comment by ChemicalXandco** [comment]
I had done something similar with #544, I'll make a PR with this feature.

**Comment by natowi** [comment]
@ChemicalXandco I think something similar is included in this PR (Task Manager): https://github.com/alicevision/meshroom/pull/612

**Comment by ChemicalXandco** [comment]
@natowi I have looked at it and I cannot find this feature.

**Comment by natowi** [comment]
@ChemicalXandco Yes, the feature to move between nodes and chunks is not included, but it can be based on the task manager.

**Comment by natowi** [comment]
The 2021.1.0 release now has a task manager

**Comment by evo-za** [comment]
I like the new task manager as a nice overview to what's happening, however it would be great if clicking on the relevant "chunk" in the task manager actually jumped to that chunk in the log. I've also noticed that pressing the down arrow to move between chunks while in the log no longer works with 2021.1.0. It briefly jumps to the next chunk and then promptly snaps back to the active chunk.
